component-samples/pom.xml

```diff
@@ -25,6 +25,7 @@
     <module>rv</module>
     <module>reseller</module>
     <module>http-device-sample</module>
+    <module>component-http-client-to0-sample</module>
```

Comment:
Once we do other rename changes, this would need this to be updated as 'to0client'